src/Agent.Plugins/PipelineArtifact/FileContainerProvider.cs

```diff
@@ -37,13 +38,15 @@ public FileContainerProvider(VssConnection connection, IAppTraceSource tracer)
             }
         }
 
-        public async Task DownloadSingleArtifactAsync(PipelineArtifactDownloadParameters downloadParameters, BuildArtifact buildArtifact, CancellationToken cancellationToken)
+        public async Task DownloadSingleArtifactAsync(PipelineArtifactDownloadParameters downloadParameters, BuildArtifact buildArtifact, CancellationToken cancellationToken, AgentTaskPluginExecutionContext context = null)
         {
+            context.Warning($"Please use Download Build Artifact task for downloading {PipelineArtifactConstants.Container} type artifact");
```

Comment:
"Container" is meaningless to people.  I think you should say Build Artifact to align to documentation.

Reply:
Yup, changed the warning message.